app/component/search/autosuggest.cjsx

```diff
@@ -60,9 +60,17 @@ class Autosuggest extends React.Component
         {displayText}
       </span>
 
+  componentDidMount: =>
+    if @refs.input.refs.input.value == ""
```

Comment:
You could check this with Hannes and see if he has ideas how to make this more clean. Problem here is that we are touching child component refs directly which is dirty...